examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1034,8 +1083,12 @@ def main(args):
         vae_path,
         subfolder="vae" if args.pretrained_vae_model_name_or_path is None else None,
         revision=args.revision,
-        variant=args.variant,
+        variant=args.variant if "playgroundai" not in args.pretrained_model_name_or_path else None,
     )
+    if "playgroundai" in args.pretrained_model_name_or_path:
+        latents_mean = torch.tensor(vae.config.latents_mean).view(1, 4, 1, 1)
+        latents_std = torch.tensor(vae.config.latents_std).view(1, 4, 1, 1)
```

Comment:
Can't this be derived from the configuration? If so, we can set a var accordingly and maybe try to avoid to use `"playgroundai"` in many places.

Reply:
+1

Reply:
Done in c8ed8af8. LMK.